liberty-ls/pom.xml

```diff
@@ -107,12 +113,13 @@
                                         <Main-Class>io.openliberty.tools.langserver.LibertyLanguageServerLauncher</Main-Class>
                                     </manifestEntries>
                                 </transformer>
-                                <transformer implementation="org.apache.maven.plugins.shade.resource.ApacheLicenseResourceTransformer"/>
-                                <transformer implementation="org.apache.maven.plugins.shade.resource.ApacheNoticeResourceTransformer">
-                                    <addHeader>false</addHeader>
-                                </transformer>
                                 <transformer implementation="org.apache.maven.plugins.shade.resource.DontIncludeResourceTransformer">
-                                    <resource>.html</resource>
+                                    <resources>
+                                        <resource>.html</resource>
+                                        <resource>META-INF/NOTICE</resource>
```

Comment:
Just wondering why you didn't include as many formats for the notice and license files here as you did for lemminx-liberty?

Reply:
I think excluding the same list of files for each makes more sense.

Reply:
ok..done